### PR TITLE
local io performance blog

### PR DESCRIPTION
**Pull request by tdthatcher**
Adds "local I/O performance" discussion from March 2022 as a blog entry.

### CODE REVIEW

File: docs/blog/2022-03-05-local-io-performance.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: Local I/O Performance on Eagle
+postdate: March 05, 2022
+layout: post
+description: A quick analysis of local I/O performance on Eagle compute nodes versus Lustre storage.
+---
+# Eagle Local I/O Performance
+
+We sometimes receive questions about disk types and I/O performance on compute nodes. Eagle has two network file systems. Qumulo provides /home and /nopt. It is NFS and is not considered a performance file system. /home has snapshots for restoration of lost data, but should not be used as a replacement for a source code repository like Git. Lustre is our performance file system, and it provides storage for the /scratch, /projects, /shared-projects and /datasets directories.
+
+Eagle also has two storage options on the compute nodes. /dev/shm, is an in-memory space (shm: shared memory), which is fast, but you need to balance its usage with your jobs memory usage as it is located directly in RAM. /tmp/scratch is physical storage. The type of storage and performance differ depending on the specific type of compute node.
```
Comment (mcfarljm) [inline docs/blog/2022-03-05-local-io-performance.md:11]:
```suggestion
Eagle also has two storage options on the compute nodes. /dev/shm is an in-memory space (shm: shared memory), which is fast, but you need to balance its usage with your job's memory usage as it is located directly in RAM. /tmp/scratch is physical storage. The type of storage and performance differ depending on the specific type of compute node. 
```

File: docs/blog/2022-03-05-local-io-performance.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Eagle also has two storage options on the compute nodes. /dev/shm, is an in-memory space (shm: shared memory), which is fast, but you need to balance its usage with your jobs memory usage as it is located directly in RAM. /tmp/scratch is physical storage. The type of storage and performance differ depending on the specific type of compute node. 
+
+If we look under Eagle's [Compute Node Hardware Details](https://www.nrel.gov/hpc/eagle-system-configuration.html) on the central [NREL HPC website](https://www.nrel.gov/hpc), there are nodes listed as having SATA drives, and nodes listed as having SSDs. Our SATA drives are still spinning disks, while SAS (serial attached SCSI) is how the SSD’s are connected to the node. We would generally expect the nodes with SSDs to perform better. Let’s test that out with a simple test.
```
Comment (mcfarljm) [inline docs/blog/2022-03-05-local-io-performance.md:13]: For consistent usage of plural acronyms:
```suggestion
If we look under Eagle's [Compute Node Hardware Details](https://www.nrel.gov/hpc/eagle-system-configuration.html) on the central [NREL HPC website](https://www.nrel.gov/hpc), there are nodes listed as having SATA drives, and nodes listed as having SSDs. Our SATA drives are still spinning disks, while SAS (serial attached SCSI) is how the SSDs are connected to the node. We would generally expect the nodes with SSDs to perform better. Let’s test that out with a simple test. 
```

**Review by mcfarljm — APPROVED** [review]
LGTM, just some minor edits.

**Review by yandthj — CHANGES_REQUESTED** [review]
Minor fix needed for consistent usage of acronyms

**Comment by tdthatcher** [comment]
fixed plural/possessive issue.

**Review by yandthj — DISMISSED** [review]

**Review by yandthj — APPROVED** [review]

**Pull request merged**